content/en/blog/_posts/2021-11-23-object-storage-support.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: blog
+title: 'Introducing COSI: Object Storage Support in Kubernetes'
+date: 2021-11-23
+slug: cosi-object-storage-support
+---
+
+**Authors:** Sidhartha Mani (MinIO)
+
+This article introduces the Container Object Storage Interface (COSI), a standard for provisioning and consuming object storage in Kubernetes. It is a pre-alpha feature in Kubernetes v1.23.
+
+## Motivation
+
+File and block storage are treated as first class citizens in the Kubernetes ecosystem via CSI. Workloads using CSI volumes enjoy the benefits of portability across vendors and across Kubernetes clusters without the need to change application manifests. An equivalent standard does not exist for Object storage.
+
+Object storage has been rising in popularity in recent years as an alternative form of storage to filesystems and block devices. Object storage paradigm promotes disaggregation of compute and storage. This is done by making data available over the network, rather than locally. Disaggregated architectures allow compute workloads to be stateless, which consequently makes them easier to manage, scale and automate.
+
+## COSI
+ 
+COSI aims to standardize consumption of object storage to provide the following benefits:
+
+
+   * Kubernetes Native - A completely kubernetes-API driven experience to provision, configure and manage buckets
+   * Self Service - A clear delineation between administration and operations (DevOps) to enable self-service capability for DevOps personnel
```

Comment:
This doesn't make sense to me. The term “DevOps” comes from dev and ops _co-operating_ (eg [10+ Deploys Per Day: Dev and Ops Cooperation at Flickr](https://www.youtube.com/watch?v=LdOe18KhtT4)), and _not_ from making the delineation rigid.

How about:
```suggestion
   * Self Service - Allows teams to manage their workload's storage themselves, without waiting on other teams to make or approve infrastructure changes.
```